src/React.Router/Content/Views/web.config.transform

```diff
@@ -0,0 +1,9 @@
+﻿<configuration>
+	<system.web.webPages.razor>
+		<pages>
+			<namespaces>
+				<add namespace="React.Router" />
+			</namespaces>
+		</pages>
+	</system.web.webPages.razor>
+</configuration>
```

Comment:
The react router htmlhelper lives in this namespace. I seem to recall being unable to use it if I didn't either have this statement in my web.config or an explicit using statement in file.
You have a similar file in the mvc4 project.